modules/ROOT/pages/lists.adoc

```diff
@@ -5,21 +5,106 @@
 :pluginname: Lists
 :plugincode: lists
 
-The `+lists+` plugin allows you to add numbered and bulleted lists to {productname}. To enable advanced lists (e.g. alpha numbered lists, square bullets) you should also enable the xref:advlist.adoc[Advanced List] (`+advlist+`) plugin.
+The `+lists+` plugin enables numbered and bulleted lists in {productname}. To enable advanced lists (for example, lists ordered by letter or unordered lists presented with square bullets) enable the xref:advlist.adoc[Advanced List] (`+advlist+`) plugin.
 
-The plugin also normalizes list behavior between browsers. Enable it if you have problems with consistency making lists.
+The `+lists+` plugin also normalizes list behavior between browsers. Enable it if you have problems with consistency making lists.
 
-== Basic setup
+It is important to note that, by default, {productname} includes both implicit and explicit list-creation user-interfaces.
+
+Adding the `lists` plugin and the `bullist` and `numlist` toolbar items to a {productname} configuration enables the explicit list-creation user-interface (UI).
+
+The implicit list-creation UI comes via xref:content-behavior-options.adoc#text_patterns[text patterns]. The default set of `text_patterns` enabled when {productname} is initialized include https://daringfireball.net/projects/markdown/syntax[Markdown]-based patterns that {productname} automatically turns into basic ordered and unordered lists.
+
+If the xref:advlist.adoc[Advanced Lists] Premium plugin is part of a {productname} configuration, the default `text_patterns` set also includes patterns that provide an implicit UI for generating the advanced ordered and unordered lists this plugin makes possible.
+
+== Basic lists UI: implicit and explicit
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'lists',
+  toolbar: 'bullist numlist',
+});
+----
+
+
+== Basic lists UI: explicit lists UI but no implicit lists UI
```

Comment:
Is there any explanation for what is meant by explicit and implicit UI? I'm not sure I understand